privaterelay/middleware.py

```diff
@@ -120,3 +133,16 @@ def immutable_file_test(self, path, url):
             return True
         else:
             return super().immutable_file_test(path, url)
+
+    def is_staticfile(self, path_info: str) -> bool:
+        """
+        Returns True if this file is served by the middleware.
+
+        This uses the logic from whitenoise.middleware.WhiteNoiseMiddleware.__call__:
+        https://github.com/evansd/whitenoise/blob/220a98894495d407424e80d85d49227a5cf97e1b/src/whitenoise/middleware.py#L117-L124
+        """
+        if self.autorefresh:
```

Comment:
(suggestion, non-blocking): Since we are using data that can come from user input I recommend using `shlex.quote`.

Reply:
No, that would change the logic from `.__call__` and break the function. See https://github.com/evansd/whitenoise/blob/220a98894495d407424e80d85d49227a5cf97e1b/src/whitenoise/middleware.py#L117-L124.